doc/source/advanced-use.rst

```diff
@@ -81,6 +81,9 @@ To clone a jail, run:
 
 :command:`iocage clone [UUID|NAME] --name [testupdate]`
 
+.. note:: The **--name** flag is optional. :command:`iocage` assigns a
+   new UUID to the clone regardless of **--name** usage.
```

Comment:
No, we don't assign a UUID if they use `--name`, if they don't we do.

Example:
`iocage create -n foo -r 11.0-RELEASE` will create a jail named `foo`.
`iocage create -r 11.0-RELEASE` will create a jail named `fd401c05-7777-44e7-9d83-289839f26351` for example.